notebook/contracts/IERC20.sol

```diff
@@ -0,0 +1,78 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Can you move your `notebook` folder into `examples/notebooks` pls?

Reply:
Unfortunately it won't work this way.

Reply:
Current tooling needs everything in a notebook folder. We can update their core repo, but I think it makes more sense to have our notebooks in a notebook folder anyways.